_posts/2018-06-01.md

```diff
@@ -0,0 +1,199 @@
+title: "Newsletter #9 - Three's company
+category: newsletter
+permalink: /news/2018/06/
+---
+
+What is Neovim?
+---------------
+
+Neovim is a fork of the venerable text-editor vim, focused on extensibility and
+usability. It is not a rewrite but a continuation and extension of Vim. Many
+clones and derivatives exist, some very clever—but none are Vim. Neovim is built
+for users who want the good parts of Vim, and more. See a list of differences
+via [:help vim-differences](https://neovim.io/doc/user/vim_diff.html).
+
+### Participating
+
+If you are interested in contributing to Neovim, visit [the github
+page](https://github.com/neovim/neovim/blob/master/CONTRIBUTING.md) for details.
+We have a great test suite and automated CI, so you can just jump in and have a
+go. Don't be afraid to open a PR to get comments or just take advantage of the
+infrastructure!
+
+To get in touch with the team, join the [gitter channel](gitter.im/neovim) (also
+accessible via [IRC](https://webchat.freenode.net/?channels=neovim)), or visit
+the [Neovim subreddit](https://www.reddit.com/r/neovim/), which many of the
+developers read. Taking part in conversations is a contribution, too!
+
+** NEW CONTRIBUTOR SHOUTOUT? Candidates: janlazo (windows support)**
+
+** OLD CONTRIBUTOR SHOUTOUT? Candidates: tjdevries, nhyoor (man man)**
+
+
+What's new
+----------
+
+The [last newsletter](https://neovim.io/news/2017/12/) covered everything up to release 0.2.2 of Neovim. Half a year has passed, and the project has been as busy as ever improving both user and developer experience. The culmination of these efforts is release 0.3, which we'd like to introduce to you:
+
+#### 0.3 for Users
+
+- A lot of work has been put into making Neovim work better with different terminal emulators, and is still ongoing (**@justinmk**, [#7664](https://github.com/neovim/neovim/pull/7664), [#7653](https://github.com/neovim/neovim/pull/7653), [#7720](https://github.com/neovim/neovim/pull/7720), [#7640](https://github.com/neovim/neovim/pull/7640), [#7624](https://github.com/neovim/neovim/pull/7624), **@florolf**, [#7676](https://github.com/neovim/neovim/pull/7676) and **@erw7** [#8408](https://github.com/neovim/neovim/pull/8408))
+
+- Linux Users:
+  - Snap packaging available (**@kalikiana**, [#7918](https://github.com/neovim/neovim/pull/7918))
+  
+- BSD Users:
+  - Support for NetBSD (**@utkarsh009**, [#8197](https://github.com/neovim/neovim/pull/8197))
+  
+- Windows Users:
+  - Neovim ships with common tools (**@equalsraf, @justinmk**, [#6105](https://github.com/neovim/neovim/pull/6105))
+  - Building with VS2017 is now supported, see [the wiki](https://github.com/neovim/neovim/wiki/Building-Neovim#windows--msvc) (**@b-r-o-c-k**, [8084](https://github.com/neovim/neovim/pull/8084))
+  - Neovim is available on [cygwin](https://github.com/cascent/neovim-cygwin)
+  
+ - Man improvements, uses lua for speedup (**@keidax**, [#7623](https://github.com/neovim/neovim/pull/7623))
```

Comment:
I don't think that his "man improvements" are relevant in this context ;)

Reply:
Justinmk wanted this added as a showcase for the lua functionality and performance.

Reply:
then I guess s/man/many/ is in order?

Reply:
perhaps improved handling of man pages? :)